CMP1903_A1_2324/Testing.cs

```diff
@@ -26,8 +26,8 @@ public void Test()
         {
             try
             {
-                Die die = new Die();
-                number = die.Roll();
+                Die Dice = new Die();
+                number = Dice.Roll();
                 Debug.Assert(number >= 1 && number <= 6);
```

Comment:
Good use of Debug.assert to text variable

Reply:
Debug.assert used correctly